unstructured/ingest/pipeline/reformat/chunking.py

```diff
@@ -107,23 +107,19 @@ def chunk(self, elements_json_file: str) -> Optional[list[Element]]:
         if self.partition_config.partition_by_api:
             return partition_via_api(
                 filename=elements_json_file,
-                # -- If api_key or api_url are None, partition_via_api will raise an error, which
-                # -- will be caught and logged by Chunker.run()
+                # -- (jennings) If api_key or api_url are None, partition_via_api will raise an
+                # -- error, which will be caught and logged by Chunker.run()
                 api_key=self.partition_config.api_key,  # type: ignore
                 api_url=self.partition_config.partition_endpoint,  # type: ignore
                 chunking_strategy=self.chunking_config.chunking_strategy,
                 combine_under_n_chars=self.chunking_config.combine_text_under_n_chars,
-                # -- These are not currently supported by the sdk and raise an error.
-                # -- They will need to be updated once we bump the client version.
-                # combine_text_under_n_chars=self.chunking_config.combine_text_under_n_chars,
-                # include_orig_elements=self.chunking_config.include_orig_elements,
+                combine_text_under_n_chars=self.chunking_config.combine_text_under_n_chars,
```

Comment:
@scanny, tagging you for your awareness. Resolves this conversation a while back: https://github.com/Unstructured-IO/unstructured/pull/2905/files#r1577236582